packages/datetime/test/dateTimePickerTests.tsx

```diff
@@ -101,6 +100,15 @@ describe("<DateTimePicker>", () => {
                 assert.isFalse(getSelectedDay().exists());
             });
         }
+
+        it("Passing an undefined value prop twice does not clear the selected date", () => {
+            const { root, getSelectedDay } = wrap(<DateTimePicker />);
+            assert.isTrue(getSelectedDay().exists());
+            root.setProps({ value: undefined });
+            assert.isNotNull(root.state("dateValue"));
+            assert.isTrue(getSelectedDay().exists());
+        });
+
```

Comment:
Delete this trailing newline

Reply:
👍  newline is legit now that there's a second test